templates/home.html

```diff
@@ -33,73 +33,101 @@ <h2>Animals in the Shelter</h2>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                    <span aria-hidden="true">&times;</span>
+                </button>
             </div>
-            <div class="form-group">
-                <label for="editAnimalName">Name:</label>
-                <input type="text" class="form-control" id="editAnimalName" required>
+            <div class="modal-body">
+                <form id="editAnimalForm">
+                    <input type="hidden" id="editAnimalId">
+                    <div class="form-group">
+                        <label for="editAnimalType">Animal Type:</label>
+                        <input type="text" class="form-control" id="editAnimalType" required>
+                    </div>
+                    <div class="form-group">
+                        <label for="editAnimalName">Name:</label>
+                        <input type="text" class="form-control" id="editAnimalName" required>
+                    </div>
+                    <div class="form-group">
+                        <label for="editAnimalBirthDate">Birth Date:</label>
+                        <input type="date" class="form-control" id="editAnimalBirthDate" required>
+                    </div>
+                    <div class="form-group">
+                    <label for="editAnimalBreed">Animal breed:</label>
+                    <input type="text" class="form-control" id="editAnimalBreed" placeholder="Animal breed" required>
+                </div>
+                <div class="form-group">
+                    <label for="editImageUrl">Image URL:</label>
+                    <input type="text" class="form-control" id="editImageUrl" placeholder="Image URL" required>
+                </div>
+                </form>
             </div>
-            <div class="form-group">
-                <label for="editAnimalBirthDate">Birth Date:</label>
-                <input type="date" class="form-control" id="editAnimalBirthDate" required>
+            <div class="modal-footer">
+                <button type="button" class="btn btn-primary" onclick="submitEditAnimal()">Save changes</button>
+                <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
             </div>
-        </form>
-      </div>
-      <div class="modal-footer">
-        <button type="button" class="btn btn-primary" onclick="submitEditAnimal()">Save changes</button>
-        <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
-      </div>
+        </div>
     </div>
-  </div>
 </div>
 
 <script>
     // Update animal
     function updateAnimal(id) {
         // Fetch the current details of the animal to pre-populate the form.
         // This assumes you have a backend route to get details of a single animal.
-        $.getJSON(`/animal/${id}`, function(data) {
+        $.getJSON(`/animal/${id}`, function (data) {
```

Comment:
I'm not FE developer, but why we need this space between function and (data)?

Reply:
It was done by the default formatter, I don't know who taught him this) I usually trust them)